supporting_content/attributes/column_naming_convention.md

```diff
@@ -16,3 +16,57 @@ Examples of column names found in provider datasets:
 - Group wanted to get naming convention adopted broadly for provider columns as well, to simplify reporting and analysis by practitioners
 - Different conventions like camel case, snake case, and title case were considered
 - Discussion included different columns naming conventions used by databases
+
+### How to handle custom columns
```

Comment:
```suggestion
### How to handle provider-defined columns
```

Reply:
I changed all the "provider" references to "custom" because there will be other tools and libraries that will generate custom columns that don't come from cloud or service providers.